test/e2e/lazy_activation_policy_test.go

```diff
@@ -197,6 +279,51 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("3. Immediate to lazy activation policy testing", func() {
+		ginkgo.JustBeforeEach(func() {
+			framework.CreatePropagationPolicy(karmadaClient, immediatePolicy)
+			waitPropagatePolicyReconciled(namespace, immediatePolicyName)
```

Comment:
may be `immediatePolicy` is redundant, we can uniform use the `policy` variable, just like:
```
ginkgo.Context("1. Policy created before resource", func() {
  ...
  ginkgo.Context("Immediate to lazy activation policy testing", func() {
    ginkgo.BeforeEach(func() {
	policy.Spec.ActivationPreference = ""
    })
    ...
  }
}
```
the Context "Propagate dependencies" is a similar examples